docs/operations/security-overview.md

```diff
@@ -264,3 +322,31 @@ As an alternative to using the basic metadata authenticator, as shown in the pre
 
 
 Congratulations, you have configured permissions for user-assigned roles in Druid!
+
+
+## Druid security trust model
+Within Druid's trust model there users can have different authorization levels:
+- Users with resource write permissions can are allowed to anything that the druid process can do.
```

Comment:
```suggestion
- Users with resource write permissions can are allowed to do anything that the druid process can do.
```

Reply:
Should `can` also be eliminated? Like, `Users with resource write permissions are allowed to do`